server/services/webhook.py

```
@@ -50,6 +50,52 @@ async def register_to_backend_dual(
+                "DELETE",
+                f"{settings.backend_url}/api/engine/register-pending",
+                headers={"Content-Type": "application/json"},
+                json={
+                    "subjectIndex": subject_index,
+                    "engineUrl": public_url,
+                    "secretKey": secret_key,
+                },
+                timeout=5.0,
+            )
+        print(f"[pending unregister] subject={subject_index} url={public_url} OK")
+    except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)